src/oceans/ui.jsx

```diff
@@ -1180,18 +1181,18 @@ class PondPanel extends React.Component {
           >
             {state.pondExplainFishSummary && (
               <div>
-                <div style={styles.pondPanelPreText}>
-                  {I18n.t('mostImportantPartsInDetermining')}
-                  {' “'}
-                  <span style={{color: colors.green}}>
-                    {state.word.toLowerCase()}
-                  </span>
-                  {'”'} {I18n.t('or')}
-                  {' “'}
-                  <span style={{color: colors.red}}>
-                    {I18n.t('notWord', {word: state.word}).toLowerCase()}
-                  </span>
-                  {'”'}.
+                <div style={styles.pondPanelPreText} id="pondTextMarkdown">
+                  <Markdown
+                    markdown={I18n.t(
+                      'mostImportantPartsInDeterminingFullSentence',
+                      {
+                        word: state.word.toLowerCase(),
+                        notWord: I18n.t('notWord', {
+                          word: state.word
+                        }).toLowerCase()
```

Comment:
Can we simplify to just substitute in `state.word` and put the negation translation into the sentence itself?

Reply:
I went back and forth on this and decided to keep it this way because the sentence is pointing out actual strings in the UI (like the buttons).

Reply:
Oh, that's a good point!